src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -84,7 +84,7 @@ private Task<DocumentDatabase> CreateDatabase(StringSegment databaseName)
                     }
                 }
 
-                ServerStore.TrackChange("Load", Constants.Database.Prefix + databaseName);
+                database.ContinueWith(_ => ServerStore.TrackChange("Load", Constants.Database.Prefix + databaseName));
```

Comment:
Nice, but let's change event type to Loaded, to avoid confusions in future. (then find usages by search for "Load" - with quotes)